### PR TITLE
added a basic profile page that renders rooms from state

### PR DESCRIPTION
**Pull request by jefQuery**
The styling is minimal.
Currently uses Mr. Brodie's roomName dummy data until the db queries are set up.
Still need to conditionally render data about each room on click.
And conditionally render either the profile button or the sign-up and login buttons.

### CODE REVIEW

**Comment by jefQuery** [comment]
[Build profile component and route to it in react router](https://trello.com/c/iT8fFdc2/104-build-profile-component-and-route-to-it-in-react-router)

**Comment by jefQuery** [comment]
[Of the buttons, render the profile button  only when logged in, and the sign-up and login buttons only when not logged in](https://trello.com/c/eXBwzUzd/111-of-the-buttons-render-the-profile-button-only-when-logged-in-and-the-sign-up-and-login-buttons-only-when-not-logged-in)

**Comment by jefQuery** [comment]
Of the buttons, render the profile button  only when logged in, and the sign-up and login buttons only when not logged in. Accomplished this by adding a Redux action path called loggedIn, that toggles true if there's a successful login or FB OAuth callback. It defaults to false.

**Comment by dbrodie122** [comment]
Looks good!  No way to logout as of yet, but that's cool.  We'll get there

File: client/styles.css [hunk 1]
```diff
@@ -13,6 +13,26 @@
 /*
 
 =====================
+PROFILE
+=====================
+
+*/
+.profile-height {
+  height: 600px;
+}
```
Comment (dbrodie122) [inline client/styles.css]: As much as possible, let's try to stick with percentages and ems to make things more responsive.  But it's not a big deal for now.

Comment (misteryeo) [inline client/styles.css]: Yeah, on that note. Noticed when we reduce the screen size, the gradient section of the page cuts off the content and we're left with a big chunk of white down below. Just something to add to backlog. I know we said we wouldn't worry about mobile responsiveness but it should be responsive for web I think. 👍

**Pull request merged**